escrow/escrow_test.go

```diff
@@ -280,3 +281,41 @@ func (suite *PaymentChannelServiceSuite) TestStartClaim() {
 	assert.Equal(suite.T(), suite.payment(), claim.Payment())
 	assert.Equal(suite.T(), []*Payment{suite.payment()}, claims)
 }
+
+func (suite *PaymentChannelServiceSuite) TestVerifyGroupId() {
+
+
+	service := NewPaymentChannelService(
```

Comment:
Test could be simplified and made more readable by copying `suite.service` and replacing only field required to replace:
```golang
service := suite.service()
service.(lockingPaymentChannelService).blockchainReader.replicaGroupID = func(channelID *big.Int) (*blockchain.MultiPartyEscrowChannel, bool, error) {
    return suite.mpeChannel(), true, nil
}
```

Reply:
Thansk @vsbogd  , fixed this